slf4j-api/src/main/java/org/slf4j/Logger.java

```diff
@@ -63,6 +70,23 @@
  */
 public interface Logger {
 
+    /**
+     * Utility method to cast a lambda to a Supplier. Best used once statically imported.
+     * <pre>
+     * import static org.slf4j.Logger.lazy;
+     *
+     * ...
+     *
+     * logger.atError().log("This is a {}", lazy(() -> "test"));
+     * </pre>
+     *
+     * @param supplier the lambda to cast
+     * @return a Supplier
+     */
+    static Supplier<?> lazy(Supplier<?> supplier) {
```

Comment:
Maybe 
```java
logger.atError().log("This is a {}", () -> "test");
```
is easier to use than 
```
logger.atError().log("This is a {}", lazy(() -> "test"));
```

I think both should be supported.
Why not?

Reply:
It does not compile

Reply:
When will this version release? slf4j community doesn't seem to be active ?